packages/blueprints-integration/src/api/showStyle.ts

```diff
@@ -169,6 +169,17 @@ export interface ShowStyleBlueprintManifest<TRawConfig = IBlueprintConfig, TProc
 		coreConfig: BlueprintConfigCoreConfig
 	) => TProcessedConfig
 
+	/**
+	 * Validate the blueprint config passed to this blueprint according to the API schema, returning a list of messages to display to the user.
+	 */
+	validateConfigFromAPI?: (context: ICommonContext, apiConfig: object) => Array<IConfigMessage>
```

Comment:
We should add to the description what the impact of the returned notes will be.
(This applies to the other methods as well)